### ISSUE TITLE
Parameter for omega to use nth-of-type

### ISSUE THREAD

**Issue by janpe** [thoughtbot/neat#318]
Would it be possible to add a parameter to have omega mixin use nth-of-type instead of nth-child? nth-child doesn't work in iOS8 and this could be used as a workaround.

**Comment by wardpenney** [comment]
Wouldn't `nth-of-type` require your sibling classes to the be the same class though? Good point about iOS though; `nth-child` isn't sustainable.

**Comment by joshuaogle** [comment]
`nth-of-type` doesn't work well in IE8 either as far as I know. There are some fancy workarounds [using adjacent selector](http://stackoverflow.com/questions/13828644/ie8-what-to-use-in-place-of-nth-of-typen), but I think the simpler solution is just to add a class to the last element like some older grid systems.

**Comment by janpe** [comment]
@wardpenney I don't see the problem with all grid items having a mutual class.

@joshuaogle I'm fine with not supporting IE8 (in most cases) but not having iOS8 support is unacceptable. Also if you're still supporting IE8, you probably won't be doing that in the not so distant future. Of course there are projects that need to have support for such browsers, but the number of those projects is greatly diminishing.

Currently I am using the following, but it's just not practical to be doing this for every single grid I have:

```
&:nth-of-type(2n) {
	@include omega;
}
&:nth-of-type(2n+1) {
	clear: both;
}
```

**Comment by joshuaogle** [comment]
@janpe I completely misread iOS8 as IE8, sorry about that. That's quite a bug. I'll keep looking into it

**Comment by admench** [comment]
+1 on this. As iOS8 doesn't support nth-child I, too am having to manually fix this in my grid layouts that use @include omega(3n) for example.

**Comment by joshuaogle** [comment]
Added a possible fix in PR in #341.
